APIs/openEO/openEO.qmd

```diff
@@ -45,7 +45,7 @@ The following limitations need to be taken into account:
 * Synchronous requests are limited to 2 concurrent requests
 * Batch jobs are limited to 2 concurrent jobs
 
-These limits are in place to avoid that individual users can overload the service, please contact support if they are problematic for your use case.
+These limits are in place to avoid that individual users can overload the service. Please contact support via [Submit a request](https://helpcenter.dataspace.copernicus.eu/hc/en-gb/requests/new){target='_blank'} if they are problematic for your use case.
```

Comment:
maybe more rephrasing of this sentence is required

Reply:
Done